diff --git a/picocolors.js b/picocolors.js
index 8b8a23e41b156f7ec0d84f6f6126385f1dd8418d..44403614be29fd63461a3f8ae915feb44d596231 100644
--- a/picocolors.js
+++ b/picocolors.js
@@ -41,6 +41,7 @@ let createColors = (enabled = isColorSupported) => {
 		inverse: init("\x1b[7m", "\x1b[27m"),
 		hidden: init("\x1b[8m", "\x1b[28m"),
 		strikethrough: init("\x1b[9m", "\x1b[29m"),
+
 		black: init("\x1b[30m", "\x1b[39m"),
 		red: init("\x1b[31m", "\x1b[39m"),
 		green: init("\x1b[32m", "\x1b[39m"),
@@ -50,6 +51,14 @@ let createColors = (enabled = isColorSupported) => {
 		cyan: init("\x1b[36m", "\x1b[39m"),
 		white: init("\x1b[37m", "\x1b[39m"),
 		gray: init("\x1b[90m", "\x1b[39m"),
+		redBright: init("\x1b[91m", "\x1b[39m"),
+		greenBright: init("\x1b[92m", "\x1b[39m"),
+		yellowBright: init("\x1b[93m", "\x1b[39m"),
+		blueBright: init("\x1b[94m", "\x1b[39m"),
+		magentaBright: init("\x1b[95m", "\x1b[39m"),
+		cyanBright: init("\x1b[96m", "\x1b[39m"),
+		whiteBright: init("\x1b[97m", "\x1b[39m"),
+
 		bgBlack: init("\x1b[40m", "\x1b[49m"),
 		bgRed: init("\x1b[41m", "\x1b[49m"),
 		bgGreen: init("\x1b[42m", "\x1b[49m"),
diff --git a/types.ts b/types.ts
index b4bacee4909e7f562fb13f89720c8ae57c4922fc..5b41ab0c5dcd754cbbc111d43fd063312554c26a 100644
--- a/types.ts
+++ b/types.ts
@@ -10,6 +10,7 @@ export interface Colors {
 	inverse: Formatter
 	hidden: Formatter
 	strikethrough: Formatter
+
 	black: Formatter
 	red: Formatter
 	green: Formatter
@@ -19,6 +20,14 @@ export interface Colors {
 	cyan: Formatter
 	white: Formatter
 	gray: Formatter
+	redBright: Formatter
+	greenBright: Formatter
+	yellowBright: Formatter
+	blueBright: Formatter
+	magentaBright: Formatter
+	cyanBright: Formatter
+	whiteBright: Formatter
+
 	bgBlack: Formatter
 	bgRed: Formatter
 	bgGreen: Formatter
